db/migrate/20160531152256_add_payment_mode_to_conferences.rb

```diff
@@ -0,0 +1,5 @@
+class AddPaymentModeToConferences < ActiveRecord::Migration
+  def change
+    add_column :conferences, :payment_mode, :string, default: 'offline'
```

Comment:
Hmm now that I think about it, "payment method" is better than payment mode :-\
http://www.google.com/trends/explore#q=%22payment%20method%22%2C%20%22payment%20mode%22&cmpt=q&tz=Etc%2FGMT-2